src/transformers/tokenization_roberta.py

```diff
@@ -69,6 +69,7 @@ class RobertaTokenizer(GPT2Tokenizer):
     vocab_files_names = VOCAB_FILES_NAMES
     pretrained_vocab_files_map = PRETRAINED_VOCAB_FILES_MAP
     max_model_input_sizes = PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES
+    n_seps_between_sentences = 2
```

Comment:
I guess this is fair. Maybe we would like to allow the user to control that easily through the init as well, as I believe RoBERTa uses a different number of separators across its fine-tuning tasks cc @julien-c.

We can leave it like this for now, though.

Reply:
yeah its defaulted in Roberta to 1